src/faq.md

```diff
@@ -116,24 +115,29 @@ literally on my first day of writing Dart code, I was productive in it."
 
 ### Q. Does Dart have reflection capabilities?
 
-We have reflection support from
+For servers and command-line scripts, we have reflection support from
 the <a href="/articles/libraries/reflection-with-mirrors">mirrors API</a>.
+We don't recommend using mirrors for web apps. The Flutter SDK [does not use
+mirrors.](https://flutter.io/faq/#does-flutter-come-with-a-reflectionmirrors-system)
 
 ### Q. Can Dart add tuples, pattern matching, non-nullable types, partial evaluation, optional semicolons, ...?
 
-The language is now at 1.x, but we anticipate further language evolution
-to occur in a standards group.  It might be able to include your feature,
-although it can't include everything.  Some features don't fit the basic nature
-of the language, and some don't play well with other features.  Simplicity is
+The language is now at 1.x, and [Dart 2.0](/dart-2.0) is in devleopment.
+We expect further language evolution to occur in a standards group.
+
+Future releases might be able to include your feature,
+although we can't include everything.
+Some features don't fit the basic nature of the language,
+and some don't play well with other features. Simplicity is
 the single most important gift we all can give to future programmers.
 
 Please look at the [list of Dart issues][issues] to see if your request is
```

Comment:
Wow, there are thousands of issues! Let's use a label to narrow it down: https://github.com/dart-lang/sdk/issues?q=is%3Aopen+is%3Aissue+label%3Aarea-language